db/queries/cards.sql

```diff
@@ -111,7 +111,7 @@ WHERE user_id = $1
     AND COALESCE(deck_id, 0) <> sqlc.arg(deck_id)::BIGINT;
 
 -- name: GetCardsInOtherDecks :many
-SELECT id, alphagram, COALESCE(deck_id, 0) as deck_id
```

Comment:
drive-by fix -- `id` does not exist, this endpoint just hadn't been used on the front-end yet